Library/ViewModels/CommentCellViewModelTests.swift

```diff
@@ -213,6 +213,40 @@ internal final class CommentCellViewModelTests: TestCase {
     }
   }
 
+  func testOutputs_replyButtonIsHidden_FeatureFlag_True_IsCreatorOrCollaborator_False() {
```

Comment:
May want to change this name to `IsNotCreatorOrCollaborator` because `IsCreatorOrCollaborator_False` is the same as `IsNotCreatorOrCollaborator_True` below. Also may want to explicitly state the `project.personalization.backing` to `nil` and `project.personalization.isBacking` to `.some(false)` to make it clear you're testing the `.memberData.permissions` as the creator/collaborator.